tests/integration/models/reddit/test_submission.py

```diff
@@ -448,6 +456,14 @@ def test_unlock(self):
         ):
             Submission(self.reddit, "4s2idz").mod.unlock()
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_unset_original_content(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+            "TestSubmissionModeration.test_unset_original_content"
+        ):
+            Submission(self.reddit, "dueqm6").mod.unset_original_content()
```

Comment:
Same comment as above regarding more comprehensive testing. Be aware that you may have issues with using the same post for both tests.